charts/akeyless-secure-remote-access/values.yaml

```diff
@@ -367,15 +370,19 @@ sshConfig:
     fsGroup: 0
     runAsUser: 0
 
-  env: []
+  env:
+    # This env variable instruct docker to start only ssh-proxy-bastion processes
+    - name: "BASTION"
+      value: "ssh-proxy"
+
   initContainer:
     image:
       repository: busybox
       tag: 1.30.1
       pullPolicy: IfNotPresent
 
   image:
-    repository: akeyless/ssh-proxy
+    repository: akeyless/zero-trust-bastion
     pullPolicy: Always
```

Comment:
pullPolicy: Always? why not ifnotpresent ???

Reply:
idk, it was like this before, didn't change it..

Reply:
@niv-ez @omriezra there is a reason for it? or can i change it to `ifnotpresent`?

Reply:
Agreed